source/mainnet/net/guides/run-node.rst

```diff
@@ -115,6 +127,7 @@ To run a node on testnet use the following configuration file and follow the ste
        ports:
        - "8889:8889"
        - "10001:10001"
+       - "20001:20001"
        volumes:
        # The node's database should be stored in a persistent volume so that it
        # survives container restart. In this case we map the **host** directory
```

Comment:
We can add to the section below "The main differences from the testnet configuration are":

- the node’s GRPC V2 listens on port 20000 instead of 20001

Reply:
@mh-concordium I'm not clear about where you want this.

Reply:
Below the config file for Testnet in the "Run a node with Docker"

![image](https://user-images.githubusercontent.com/98821384/196179513-18d9128a-9eed-453d-97cb-7648f99e6057.png)